paddle/fluid/framework/details/all_reduce_op_handle.cc

```diff
@@ -28,7 +28,7 @@
 // asynchronous nccl allreduce or synchronous issue:
 // https://github.com/PaddlePaddle/Paddle/issues/15049
 DEFINE_bool(
-    sync_nccl_allreduce, false,
+    sync_nccl_allreduce, true,
```

Comment:
It's not useful now:
With sync:
![image](https://user-images.githubusercontent.com/10721757/55152463-6ba83300-518b-11e9-8d27-2a3d61646e1f.png)

Withou sync:
![image](https://user-images.githubusercontent.com/10721757/55152483-782c8b80-518b-11e9-9a67-6e60fbd1b9c6.png)